app/models/setting.rb

```diff
@@ -30,16 +30,20 @@ def validate(record)
     end
   end
 
+  def self.blank_attrs
```

Comment:
Can't we make this something like `blank_attr?(name)` because I feel like that would be more efficient and perhaps easier to read as well.

Reply:
Indeed ! Makes sense, great suggestion!